<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>411</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Dados da Lista</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAluno">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>111</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="fotos.qrc">:/new/prefix1/myPhoto.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelPuc">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="fotos.qrc">:/new/prefix1/puc-goias.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>401</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <italic>false</italic>
             <bold>true</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>Inserir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonInserirInicio">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Inicio</string>
           </property>
           <property name="icon">
            <iconset resource="fotos.qrc">
             <normaloff>:/new/prefix1/inserir.png</normaloff>:/new/prefix1/inserir.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonInserirFim">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Fim</string>
           </property>
           <property name="icon">
            <iconset resource="fotos.qrc">
             <normaloff>:/new/prefix1/inserir.png</normaloff>:/new/prefix1/inserir.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonInserirPosicao">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Posicao</string>
           </property>
           <property name="icon">
            <iconset resource="fotos.qrc">
             <normaloff>:/new/prefix1/inserir.png</normaloff>:/new/prefix1/inserir.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <italic>false</italic>
             <bold>true</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>Acessar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonAcessarInicio">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Inicio</string>
           </property>
           <property name="icon">
            <iconset resource="fotos.qrc">
             <normaloff>:/new/prefix1/lupa.png</normaloff>:/new/prefix1/lupa.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonAcessarFim">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Fim</string>
           </property>
           <property name="icon">
            <iconset resource="fotos.qrc">
             <normaloff>:/new/prefix1/lupa.png</normaloff>:/new/prefix1/lupa.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonAcessarPosicao">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Posicao</string>
           </property>
           <property name="icon">
            <iconset resource="fotos.qrc">
             <normaloff>:/new/prefix1/lupa.png</normaloff>:/new/prefix1/lupa.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <italic>false</italic>
             <bold>true</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>Retirar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonRetirarInicio">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Inicio</string>
           </property>
           <property name="icon">
            <iconset resource="fotos.qrc">
             <normaloff>:/new/prefix1/saida.png</normaloff>:/new/prefix1/saida.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonRetirarFim">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Fim</string>
           </property>
           <property name="icon">
            <iconset resource="fotos.qrc">
             <normaloff>:/new/prefix1/saida.png</normaloff>:/new/prefix1/saida.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonRetirarPosicao">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Posicao</string>
           </property>
           <property name="icon">
            <iconset resource="fotos.qrc">
             <normaloff>:/new/prefix1/saida.png</normaloff>:/new/prefix1/saida.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ordenado</string>
       </property>
       <property name="icon">
        <iconset resource="fotos.qrc">
         <normaloff>:/new/prefix1/inserir.png</normaloff>:/new/prefix1/inserir.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <italic>false</italic>
             <bold>true</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>Valor:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditValor"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <italic>false</italic>
             <bold>true</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>Posicao:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditPosicao"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>270</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Qtd: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditQtd"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLLDE">
    <property name="title">
     <string>LLDE</string>
    </property>
   </widget>
   <addaction name="menuLLDE"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="fotos.qrc"/>
 </resources>
 <connections/>
</ui>
